### PR TITLE
Polish k8s_instance_manager.

### PR DESCRIPTION
**Pull request by brightcoder01**
- Polish the current code of k8s_instance_manager at first. And then we will do some refactor in the next step.
- Do function renaming; add more descriptions; extract utility functions to make the function body short.

### CODE REVIEW

**Review by workingloong — DISMISSED** [review]
Excellent!

File: elasticdl/python/master/k8s_instance_manager.py [hunk 1]
```diff
@@ -51,6 +58,17 @@ def _parse_worker_pod_priority(num_workers, worker_pod_priority):
     return res
 
 
+def _is_pod_oom_killed(evt_obj):
```
Comment (skydoorkai) [inline elasticdl/python/master/k8s_instance_manager.py]: This is not `_is_pod_oom_killed`, but `killed_but_not_oomed`.

Comment (brightcoder01) [reply elasticdl/python/master/k8s_instance_manager.py]: Good catch, the method name is updated.

File: elasticdl/python/master/k8s_instance_manager.py [hunk 2]
```diff
@@ -26,6 +26,17 @@
 _SERVICE_ADDR_SEP = ","
 
 
+def _get_addrs(num_addrs, addr_get_fn):
+    """
+    Get `num_addrs` addresses and the concatenate
```
Comment (workingloong) [inline elasticdl/python/master/k8s_instance_manager.py]: and then ?

Comment (brightcoder01) [reply elasticdl/python/master/k8s_instance_manager.py]: Good catch, updated.

**Review by workingloong — APPROVED** [review]

**Review by skydoorkai — APPROVED** [review]

**Pull request merged**